src/main/java/io/github/nucleuspowered/nucleus/configurate/typeserialisers/DelayedTimeValueSerialiser.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * This file is part of Nucleus, licensed under the MIT License (MIT). See the LICENSE.txt file
+ * at the root of this project for more details.
+ */
+package io.github.nucleuspowered.nucleus.configurate.typeserialisers;
+
+import com.google.common.reflect.TypeToken;
+import com.cronutils.model.Cron;
+import com.cronutils.model.CronType;
+import com.cronutils.model.definition.CronDefinitionBuilder;
+import com.cronutils.parser.CronParser;
+import io.github.nucleuspowered.nucleus.util.ClockTime;
+import io.github.nucleuspowered.nucleus.util.DelayedTimeType;
+import io.github.nucleuspowered.nucleus.util.DelayedTimeValue;
+import io.github.nucleuspowered.nucleus.util.WhenTime;
+import ninja.leaping.configurate.ConfigurationNode;
+import ninja.leaping.configurate.SimpleConfigurationNode;
+import ninja.leaping.configurate.objectmapping.ObjectMappingException;
+import ninja.leaping.configurate.objectmapping.serialize.TypeSerializer;
+
+import java.util.Map;
+
+/**
+ * Manages serialization of a DelayedTimeValue
+ */
+public class DelayedTimeValueSerialiser implements TypeSerializer<DelayedTimeValue> {
+
+    private CronParser unixCronParser;
+    private CronParser quartzCronParser;
+    private CronParser cronjCronParser;
+
+    /**
+     * Constructs a DelayedTimeValueSerialiser, initializes the cron parsers
+     */
+    public DelayedTimeValueSerialiser() {
+        this.unixCronParser = new CronParser(CronDefinitionBuilder.instanceDefinitionFor(CronType.UNIX));
+        this.quartzCronParser = new CronParser(CronDefinitionBuilder.instanceDefinitionFor(CronType.QUARTZ));
+        this.cronjCronParser = new CronParser(CronDefinitionBuilder.instanceDefinitionFor(CronType.CRON4J));
+    }
+
+    @Override
+    public DelayedTimeValue deserialize(TypeToken<?> type, ConfigurationNode value) throws ObjectMappingException {
+        if (value.getValue() instanceof String) {
+            String node = value.getString();
+            try {
```

Comment:
That's true. Admittedly I was unhappy duplicating, but was unsure of where to put it. However, I like your suggestion so I'll go ahead and move it there.